### DISCUSSION TITLE
When monitor is off: Crash "Calculated bogus refresh interval"

### DISCUSSION THREAD

**Discussion by mabahj** [guysoft/FullPageOS#572] [Q&A]
When I turn off the monitor (using xrandr) crashes continuously and is continuously restarted. It keeps my raspberry very busy and is fills up logs (and wears on my SDCard).

This may not be a bug worth fixing, it is not blocking in any way. But I'm wondering if anyone has any tips on how I can maybe pause this while the monitor is off? Or if there is something else I'm doing wrong? (And I find it strange that I do not find any reports on this...?)

(guysoft: Thank you very much for a nice tool/distro! Appreciated!)

#### What were you doing?
I turn off the screen (at night) with this command:
`xrandr --display :0 --output HDMI-2 --off`

#### What did you expect to happen?
I would expect FullPageOS to be happy with that

#### What happened instead?
Chrome crashes continuously and is restarted every second.

#### Was there an error message displayed? What did it say?
From /tmp/crashlog:
```
Restarting at Thu 15 Feb 20:24:24 CET 2024 after a reported crash. Logline: [1100:1100:0215/202422.203613:ERROR:sync_control_vsync_provider.cc(139)] Calculated bogus refresh interval=0.999409 s, last_timebase_=47451007251 bogo-microseconds, timebase=47452006660 bogo-microseconds, last_media_stream_counter_=2834959, media_stream_counter=2834960
Restarting at Thu 15 Feb 20:24:25 CET 2024 after a reported crash. Logline: [1100:1100:0215/202423.199223:ERROR:sync_control_vsync_provider.cc(139)] Calculated bogus refresh interval=0.995552 s, last_timebase_=47452006660 bogo-microseconds, timebase=47453002212 bogo-microseconds, last_media_stream_counter_=2834960, media_stream_counter=2834961
Restarting at Thu 15 Feb 20:24:27 CET 2024 after a reported crash. Logline: [1100:1100:0215/202424.200859:ERROR:sync_control_vsync_provider.cc(139)] Calculated bogus refresh interval=1.0015 s, last_timebase_=47453002212 bogo-microseconds, timebase=47454003708 bogo-microseconds, last_media_stream_counter_=2834961, media_stream_counter=2834962
Restarting at Thu 15 Feb 20:24:28 CET 2024 after a reported crash. Logline: [1100:1100:0215/202425.200417:ERROR:sync_control_vsync_provider.cc(139)] Calculated bogus refresh interval=0.999751 s, last_timebase_=47454003708 bogo-microseconds, timebase=47455003459 bogo-microseconds, last_media_stream_counter_=2834962, media_stream_counter=2834963
(and so on)
```

#### Version of FullPageOS?
FullPageOS 0.13.0
(Installed using Raspberry Pi Imager)

#### Screenshot(s) showing the problem:
N/A

#### If you are building FullPageOS - provide a build.log that is created for the build
N/A

**Comment by guysoft** [comment]
Thanks, if you want we can move this in to the discussions, there is a forum here now.

**Comment by mabahj** [comment]
I did not know. It would be great if you could move it. Thank you.
